### PR TITLE
Adding a process for requesting feedback

### PR DESCRIPTION
**Pull request by yanokwa**

### CODE REVIEW

File: TSC-1/README.md [hunk 1]
```diff
@@ -64,6 +64,18 @@ The typical process is:
 - The TSC discusses and refines specifications during its meetings
 - Accepted specifications are placed on the [roadmap](https://github.com/opendatakit/roadmap/projects/1)
 
+## Requesting feedback
```
Comment (smoyth) [inline TSC-1/README.md]: Can we please call this `Lazy consensus guidelines`

The title 'Requesting feedback' implies the audience of this document is committers/community members and not TSC members, which is inconsistent with other sections I believe. It also does nothing to link this section to the "establish appropriate guidelines" section in the TSC-GOVERNANCE.

File: TSC-1/README.md [hunk 2]
```diff
@@ -64,6 +64,18 @@ The typical process is:
...
+
+The TSC wants to provide thoughtful asynchronous feedback to committers and other TSC members, but we also don't want to block work.
```
Comment (smoyth) [inline TSC-1/README.md]: Can we please change this to:

The TSC-GOVERNANCE document specifies: "For internal project decisions, Committers shall operate under Lazy Consensus. The TSC shall establish appropriate guidelines for implementing Lazy Consensus (e.g., expected notification and review time periods)." This section lays out those guidelines. Our goal is to allow sufficient time for thoughtful, asynchronous deliberation and feedback by the TSC without unduly impeding the work of those in need of a decision.

File: TSC-1/README.md [hunk 3]
```diff
@@ -64,6 +64,18 @@ The typical process is:
...
+
+If feedback from the TSC is needed, it should be in the form of a proposal from a committer or TSC member and it should have a deadline included. The TSC must be notified by being @mentioned in a post on the forum. Everyone on the forum can @mention the TSC and the notifications will also appear in #tsc-1-pulse on Slack.
```
Comment (smoyth) [inline TSC-1/README.md]: Change "If feedback from the TSC is needed, it should be in the form of a proposal from" to "If a decision from the TSC is called for in a given matter, a proposal should be made by a"

File: TSC-1/README.md [hunk 4]
```diff
@@ -64,6 +64,18 @@ The typical process is:
...
+
+TSC members have at least 72 hours (3 days) to object to proposals. Controversial or breaking changes (e.g., spec additions, feature removal, governance changes) require at least 120 hours (5 days) to object. TSC members can request more time (typically no more than a doubling of the initial time).
+
+TSC members should acknowledge reading the proposal (preferably a comment) so there is a visible record of their approval. If there are no objections, the proposal can proceed. If there are objections, there should be discussion until there are no more objections or the proposal is withdrawn or rejected.
```
Comment (smoyth) [inline TSC-1/README.md]: "preferably with a comment"

File: TSC-1/README.md [hunk 5]
```diff
@@ -64,6 +64,18 @@ The typical process is:
...
+
+This process does not apply to GitHub PRs (e.g., for the website) or Slack conversations or emails. This is by design. We want to have a single place (the forum) where all TSC members are expected to respond.
```
Comment (smoyth) [inline TSC-1/README.md]: Change "This process does not apply to ..." to "Asynchronous decisions of the TSC may be made only via the forum, and not"

**Comment by yanokwa** [comment]
@smoyth Changes made and force pushed. If you are happy with them, please approve!

**Pull request merged**